OrbitGl/TimeGraph.h

```diff
@@ -59,12 +59,14 @@ class TimeGraph {
   void SetMinMax(double a_MinTimeUs, double a_MaxTimeUs);
   void PanTime(int a_InitialX, int a_CurrentX, int a_Width,
                double a_InitialTime);
+  void HorizontallyScrollIntoView(const TextBox* text_box);
   void VerticallyScrollIntoView(const TextBox* text_box);
   double GetTime(double a_Ratio);
   double GetTimeIntervalMicro(double a_Ratio);
-  void Select(const TextBox* a_TextBox) { SelectRight(a_TextBox); }
-  void SelectLeft(const TextBox* a_TextBox);
-  void SelectRight(const TextBox* a_TextBox);
+  void Select(const TextBox* a_TextBox);
+  enum JumpScope { SameThread, SameFunction, SameThreadSameFunction };
```

Comment:
Just one more thing here: Enums should use the constant naming style, that is, 

enum ... {
  kSameThread, 
  kSameFunction, 
  ...
};

Reply:
Thanks!